datasets/modeled/industry/dimensions/subsectors.csv

```diff
@@ -1,9 +1,15 @@
 ﻿id,name
-naics_3273,Cement and Concrete
-naics_3253,Pesticides and Fertilizer
-naics_3311,Iron and Steel
-naics_311_312,Food and Beverage
-naics_325_other,Chemicals
-naics_3241,Petroleum Refineries
-naics_322,Pulp and Paper
-naics_3_other,Other Manufacturing
\ No newline at end of file
+cement_concrete,Cement and Concrete
+chemical,Chemicals
+chemical_other,Chemicals
+construction,Construction
+datacenter,Data Centers
+food_bev,Food and Beverage
+iron_steel,Iron and Steel
+iron_steel_other,Iron and Steel
+mining,Mining
+other,Other Industry
+pesticides_fertilizer,Pesticides and Fertilizer
+pet_refinery,Petroleum Refineries
+pulp_paper,Pulp and Paper
+water_treatment,Water Supply and Treatment

```

Comment:
@ashreeta, it is my understanding that, like data centers, EIA classifies water treatment as commercial. So we should pull this subsector into our commercial residuals calculation, just like data centers.